File: PHP Word Examples/WordDocumentApiDemo/getDocumentChildObjects.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\WordDocumentApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new WordDocumentApi($configuration);

$name = "Template.docx";
$folder = "DocDocument";
$storage = null;
$password = null;
$result = $apiInstance->getDocumentChildObjects($name,$folder, $storage, $password);

File: Python Word Examples/WordDocumentApiDemo/getDocumentChildObjects.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.WordDocumentApi(configuration)

file_name = 'Template.docx'
extra = {'folder': "DocDocument"}
response = api.get_document_child_objects(file_name, **extra)